src/services/tx/txSender.ts

```diff
@@ -22,35 +22,34 @@ import type { Web3Provider } from '@ethersproject/providers'
 import type { ContractTransaction } from 'ethers'
 import { ethers } from 'ethers'
 import type { SpendingLimitTxParams } from '@/components/tx/modals/TokenTransferModal/ReviewSpendingLimitTx'
-import { getSpendingLimitContract } from '@/services/contracts/spendingLimitContracts'
+import { getSpendingLimitContract, getSpendingLimitModuleAddress } from '@/services/contracts/spendingLimitContracts'
 import EthersAdapter from '@gnosis.pm/safe-ethers-lib'
 import { Errors, logError } from '@/services/exceptions'
 import { EMPTY_DATA } from '@gnosis.pm/safe-core-sdk/dist/src/utils/constants'
+import { useCallback } from 'react'
+import type { SpendingLimitState } from '@/store/spendingLimitsSlice'
+import { createAddDelegateTx, createResetAllowanceTx, createSetAllowanceTx } from '@/services/tx/spendingLimitParams'
+import { parseUnits } from '@ethersproject/units'
+import { currentMinutes } from '@/utils/date'
+import type { NewSpendingLimitData } from '@/components/settings/SpendingLimits/NewSpendingLimit/steps/ReviewSpendingLimit'
 
-const getAndValidateSafeSDK = (): Safe => {
-  const safeSDK = getSafeSDK()
-  if (!safeSDK) {
-    throw new Error('The Safe SDK could not be initialized. Please be aware that we only support v1.1.1 Safes and up.')
-  }
-  return safeSDK
-}
+const SDK_NOT_FOUND_ERROR =
+  'The Safe SDK could not be initialized. Please be aware that we only support v1.1.1 Safes and up.'
```

Comment:
Since you're touching this, please remove "Please be aware that we only support v1.1.1 Safes and up.".

Reply:
Should we already remove it or wait until after it is implemented in the core sdk?